ui/raidboss/data/02-arr/dungeon/haukke_manor.js

```diff
@@ -0,0 +1,81 @@
+
+
+[{
+  zoneId: ZoneId.HaukkeManor,
+  triggers: [
+    {
+      // Dark Mist Stun with Suppression for duplicate messages
+      id: 'Haukke Normal Dark Mist Stun',
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: ['Manor Maidservant', 'Manor Claviger', 'Lady Amandine'] }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      suppressSeconds: 2,
+      response: Responses.stun('info'),
+    },
+    {
+      // Soul Drain
+      id: 'Haukke Normal Steward Soul Drain Stun',
+      netRegex: NetRegexes.startsUsing({ id: '35C', source: 'Manor Steward' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Amandine Specific Dark Mist for Classes Without Stuns
+      // Particle and spell effects in the room can make this Dark Mist hard to see.
+      // Callout ensures dodge.
+      id: 'Haukke Normal Amandine Dark Mist Dodge',
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: 'Lady Amandine' }),
+      condition: function(data, matches) {
+        return !data.CanStun();
+      },
+      response: Responses.outOfMelee('alert'),
+    },
+    {
+      // Void Fire 3
+      id: 'Haukke Normal Amandine Void Fire III',
+      netRegex: NetRegexes.startsUsing({ id: '356', source: 'Lady Amandine' }),
+      condition: function(data, matches) {
+        return data.CanSilence();
+      },
+      response: Responses.interrupt('info'),
+    },
+    {
+      // Void Thunder 3
+      id: 'Haukke Normal Amandine Void Thunder III',
+      netRegex: NetRegexes.startsUsing({ id: '358', source: 'Manor Claviger' }),
```

Comment:
Should this be Lady Amandine?

Also, should trigger be `Responses.getBehind()`? Am I misremembering, or can you line of sight Thunder III like you can do for Slipstream?

Reply:
Amandine tracks her target unfortunately with Void Thunder. You are correct it is Amandine